src/api/app/views/webui/users/tasks/index.html.haml

```diff
@@ -1,85 +1,78 @@
-                %td
-                  - pi[:issues].each do |i|
-                    = link_to i[:label], i[:url], title: i[:summary]
-                    ,
-                %td= link_to(sprite_tag('monitor', title: 'Monitor'), project_monitor_path(pi[:package][:project], pkgname: pi[:package][:name]))
+  .card-body
+    .tab-content#reviews-in
+      = render(partial: 'webui2/shared/requests_table', locals: { id: 'reviews_in_table', source_url: user_requests_path(User.session!),
+               page_length: 10 })
 
-  = javascript_tag do
-    :plain
-      $(function() {
-        $('#open-patchinfos-table').dataTable();
-      });
-- content_for :ready_function do
-  $( "#reviews" ).tabs();
-  $( "#requests" ).tabs();
-  $( "#patchinfos" ).tabs();
+.card.mt-3#requests
+  .bg-light
+    %ul.nav.nav-tabs.pt-2.px-3.flex-nowrap.collapsible{ 'role': 'tablist' }
+      %li.nav-item
+        - cache "#{User.session!.cache_key}-requests-in" do
+          %a.nav-link.text-nowrap.active#requests-in-tab{ href: '#requests-in', title: "Requests that #{User.session!} has to merge",
+            'data-toggle': 'tab', 'aria-controls': 'requests-in-tab', 'aria-selected': 'true', role: 'tab' }
+            Incoming Requests
+            %span.badge.badge-primary
+              = User.session!.incoming_requests.count
+      %li.nav-item
+        %a.nav-link.text-nowrap#requests-out-tab{ href: '#requests-out', title: "Requests that #{User.session!} has sent",
+          'data-toggle': 'tab', 'aria-controls': 'requests-out-tab', 'aria-selected': 'false', role: 'tab' }
+          Outgoing Requests
+          %span.badge.badge-primary
+            = User.session!.outgoing_requests.count
+      %li.nav-item
+        %a.nav-link.text-nowrap#requests-declined-tab{ href: '#requests-declined', title: "Requests from #{User.session!} that are declined",
+          'data-toggle': 'tab', 'aria-controls': 'requests-declined-tab', 'aria-selected': 'false', role: 'tab' }
+          Declined Requests
+          %span.badge.badge-primary
+            = User.session!.declined_requests.count
+      %li.nav-item
+        %a.nav-link.text-nowrap#all-requests-tab{ href: '#all-requests', title: "All Requests from #{User.session!}",
+          'data-toggle': 'tab', 'aria-controls': 'all-requests-tab', 'aria-selected': 'false', role: 'tab' }
+          All Requests
+          %span.badge.badge-primary
+            = User.session!.requests.count
+      %li.nav-item.dropdown
+        %a.nav-link.dropdown-toggle{ href: '#', 'data-toggle': 'dropdown', 'role': 'button', 'aria-expanded': 'false', 'aria-haspopup': 'true' }
+        .dropdown-menu.dropdown-menu-right
+  .card-body
+    .tab-content
+      .tab-pane.fade.show.active#requests-in{ role: 'tabpanel', 'aria-labelledby': 'requests-in-tab' }
+        = render(partial: 'webui2/shared/requests_table', locals: { id: 'requests_in_table', source_url: user_requests_path(User.session!),
```

Comment:
Please, use the partials that are into `webui/shared`

Reply:
done